master/pkg/model/environment_config.go

```diff
@@ -153,12 +151,7 @@ func (r *RuntimeItems) For(deviceType device.Type) []string {
 
 // Validate implements the check.Validatable interface.
 func (e Environment) Validate() []error {
-	err := validatePodSpec(e.PodSpec)
-	// Slurm and PodSpec are different targets, so no need to merge results
-	if err != nil {
-		return err
-	}
-	return ValidateSlurm(e.Slurm)
```

Comment:
did this validation land anywhere else? should we at least make a ticket for reinstating it?

Reply:
ValidateSlurm() continues to be called explicitly from the Dispatcher RM code. FYI, I wasn't able to determine that Environment.Validate() was ever called, least-ways, a break point there in the master context never fired)